loopchain/channel/channel_inner_service.py

```diff
@@ -776,9 +776,10 @@ def vote_unconfirmed_block(self, vote_dumped: str) -> None:
                 f"Peer vote to: {vote.block_height}({vote_round}) {vote_block_hash} from {voter}"
             )
 
-            if is_version_1_0 and self._channel_service.consensus_runner:
-                util.logger.notice(f'loopchain 3.x has event_system!')
-                self._channel_service.consensus_runner.receive_vote(vote)
+            if is_version_1_0:
```

Comment:
I suspect code rollback by rebase. Is it intended?

Reply:
yes, If block is version 1.0 and self._channel_service don't work, then It will work on sour code about loopchain 2.x version.